vulcan/lib/server/models/workflow.rb

```diff
@@ -38,24 +38,36 @@ def as_steps_json(name)
         }
       end
 
+      def step_key(step)
```

Comment:
Super minor nit -- I believe this step variable is actual a step_input, specifically, right?  Super minor.